backend/tournesol/tests/test_suggestions.py

```diff
@@ -271,6 +288,24 @@ def setUp(self):
             ),
         ]
 
+        for i, va in enumerate(self.videos[1:]):
+            for j, vb in enumerate(self.videos[i+2:]):
+                self.comparisons.append(ComparisonFactory(
+                    user=self.sparsity_comparison_user,
+                    entity_1=va,
+                    entity_2=vb,
+                    duration_ms=501 + i * 20 + j,
+                    datetime_lastedit=now + datetime.timedelta(minutes=2),
+                ))
+
+        self.comparisons.append(ComparisonFactory(
+            user=self.sparsity_comparison_user,
+            entity_1=self.videos[0],
+            entity_2=self.videos[4],
+            duration_ms=501,
+            datetime_lastedit=now + datetime.timedelta(minutes=2),
+        ))
```

Comment:
What's the intention behind this setup? It's not clear to me at first glance, a comment would be welcome.

(`duration` and `datetime_lastedit` could probably be omitted as they are optional in the factory).